### PR TITLE
Dp 7336 remove gs

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/app/App.jsx [hunk 1]
```diff
@@ -113,7 +114,7 @@ const App = ({
   `
 
   function AppWrapper({ children }) {
-    return homePage || editorialPage ? (
+    return homePage || editorialPage || contentPage ? (
```
Comment (royderks) [inline src/app/App.jsx]: The condition for `hasMaxwidth` and the return is the same

File: modules/page/page-config.constant.js [hunk 2]
```diff
@@ -1,15 +1,7 @@
...
     metadata: 'Actualiteit',
-    news: 'Nieuws',
-    beleid: 'Beleid',
-    help: 'Help',
-    snelwegwijs: 'Bediening portaal',
-    apis: "API's gebruiken",
-    proclaimer: 'Proclaimer',
     inloggen: 'Na inloggen',
-    statistieken: 'Statistieken',
     verplaatst: 'Pagina verplaatst',
```
Comment (royderks) [inline modules/page/page-config.constant.js]: Where are these still used? Couldn't find anything for the `nietgevonden` constant

File: src/app/pages/ContentPage.jsx [hunk 3]
```diff
@@ -1,60 +1,36 @@
...
-            interpolateBindings={{
-              name: templateName,
-              type,
-              item,
-            }}
-          />
-          <div className="u-col-sm--offset-1 u-col-sm--7">
-            <ShareBar />
-          </div>
+const StyledRow = styled(Row)``
```
Comment (royderks) [inline src/app/pages/ContentPage.jsx]: Do we need a `StyledRow` here?

Comment (aditudorache) [reply src/app/pages/ContentPage.jsx]: not anymore: removed

File: src/app/App.jsx (hunk 1 above)
Comment (aditudorache) [reply src/app/App.jsx]: sure! thanks!

File: modules/page/page-config.constant.js (hunk 2 above)
Comment (aditudorache) [reply modules/page/page-config.constant.js]: I have checked and are not used, I've removed them

**Pull request merged**